src/qt/trafficgraphwidget.cpp

```diff
@@ -47,13 +47,14 @@ int TrafficGraphWidget::getGraphRangeMins() const
 
 void TrafficGraphWidget::paintPath(QPainterPath &path, QQueue<float> &samples)
 {
-    int h = height() - YMARGIN * 2, w = width() - XMARGIN * 2;
-    int sampleCount = samples.size(), x = XMARGIN + w, y;
+    int sampleCount = samples.size();
     if(sampleCount > 0) {
+        int w = width() - XMARGIN * 2;
+        int x = XMARGIN + w, h = height() - YMARGIN * 2;
```

Comment:
a super-nit, but I preferred `w` and `h` on the same line as before, because of the symmetry.